src/components/IOUConfirmationList.js

```diff
@@ -79,6 +84,8 @@ const defaultProps = {
     comment: '',
 };
 
+const MINIMUM_BOTTOM_OFFSET = 240;
```

Comment:
One last request, could you add a comment above this const explaining why it needs to be 240. Thanks

Reply:
@Julesssss Done!

Reply:
This is a really small thing, but could you make it two lines instead? 😀

Reply:
@Julesssss Haha done! I got a little overboard with the explanation. By any chance, did you find time to test the implementation itself as well?

Reply:
Just tested, leaving another note elsewhere